guide-us-english.tex

```diff
@@ -228,7 +228,7 @@ \subsubsection{EAT Backlog \& Responsibilities}
 
 \subsection{Executive MetaScrum Team (EMT)}
 
-The Product Owner Teams enable a network design of Product Owners which is infinitely scalable with their associated SoS. The Product Owner Team for the entire agile organization meets with key stakeholders at the \textbf{Executive MetaScrum}. The EMT owns the organizational vision and sets the strategic priorities, aligning all the teams around common goals.
+The \textbf{Executive MetaScrum Team (EMT)} fulfills the Product Owner role for the entire agile organization. This leadership team owns the organizational vision and sets the strategic priorities, aligning all the teams around common goals. The Product Owner Teams enable a network design of Product Owners which is infinitely scalable with their associated SoS. The Product Owner Team for the entire agile organization meets with key stakeholders at the \textbf{MetaScrum Event} (below).
 
```

Comment:
I wouldn't accept this one in this wording, although I do agree with bringing things in line. The EMS is not just the Product Owner role, it's more than that. Needs some discussion.